api/models/Post.js

```diff
@@ -707,7 +707,7 @@ module.exports = bookshelf.Model.extend(Object.assign({
             title: post.summary(),
             type: post.get('type'),
             url: Frontend.Route.post(post),
-            groups: post.relations.groups.map(g => ({ id: g.id, name: g.get('name'), url: Frontend.Route.group(g), postUrl: Frontend.Route.post(post, g) })),
+            groups: post.relations.groups.map(g => ({ id: g.id, name: g.get('name'), url: Frontend.Route.group(g), postUrl: Frontend.Route.post(post, g) })), // This is the thing that is probably borking the zapier links
```

Comment:
I have a fix for the zapper links ready to go

Reply:
ok, yeah I was just 'in the area'... I guess if folks are getting URLs from zapier about chat posts they are going to be still be weird from a UX perspective... Do we want to fix those in the same way as the other chat post notifications now?

Reply:
Yes probably